jupyter_core/troubleshoot.py

```diff
@@ -74,16 +74,20 @@ def main():
     print('\n' + 'platform.platform():')
     print('\t' + environment_data['platform'])
 
-    print('\n' + 'which -a jupyter:')
-    print('\t' + environment_data['which'])
+    if environment_data['which']:
+        print('\n' + 'which -a jupyter:')
```

Comment:
There is a `where` command we could use on Windows, or `shutil.which()`. I don't think either of them have an equivalent of the `-a` flag, though.